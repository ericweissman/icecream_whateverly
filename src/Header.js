import React, { Component } from 'react';
import './css/Main.scss';
import Search from './Search.js';
import SearchByParlor from './SearchByParlor';


class Header extends Component {
  constructor() {
    super();
<<<<<<< HEAD
  }
  render() {
    return (
      <header className="header">
        <div className="header-svg-cont">
          <img className="header-svg" src={require('./css/images/ice-cream.svg')}/>
        </div>
        <div className="header-search-cont">
        <header className="title">iScream</header>
          <Search searchFor={this.props.searchFor}/>
          <SearchByParlor parlors={this.props.parlors} searchParlor={this.props.searchParlor}/>
        </div>
      </header>
    );
  }
=======
  }
  render() {
    return (
      <header className="header">
        <div className="header-search-cont">
          <img className="header-svg" src={require('./css/images/ice-cream.svg')}/>
          <header className="title">iScream</header>
          <Search searchFor={this.props.searchFor}/>
          <SearchByParlor parlors={this.props.parlors} searchParlor={this.props.searchParlor}/>
        </div>
      </header>
    );
  }
>>>>>>> 2a92d4c6f21b849ce00ce5bacc1c8b106ee21560
}



export default Header;