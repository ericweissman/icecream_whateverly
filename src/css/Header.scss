@import url('https://fonts.googleapis.com/css?family=Lobster');

.header {
    display: flex;
    flex-direction: column;
    font-size: 50px;
    display: flex;
    justify-content: center;
    align-items: center;
    background-color: aquamarine;
    padding: 50px;
}

.title {
    font-family: 'Lobster', cursive;
    font-size: 75px;
}
