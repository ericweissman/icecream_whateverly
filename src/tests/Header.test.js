import React from 'react';
import Header from '../Header.js';
import { shallow } from 'enzyme';

const searchForMock = jest.fn();

describe('Header', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Header searchFor={searchForMock} />
    );
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
})