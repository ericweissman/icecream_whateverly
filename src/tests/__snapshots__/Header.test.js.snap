// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with all data passed in correctly 1`] = `
<header
  className="header"
>
  <div
    className="header-svg-cont"
  >
    <img
      className="header-svg"
      src={
        Object {
          "ReactComponent": [Function],
          "__esModule": true,
          "default": "ice-cream.svg",
        }
      }
    />
  </div>
  <div
    className="header-search-cont"
  >
    <header
      className="title"
    >
      iScream
    </header>
    <Search
      searchFor={[MockFunction]}
    />
    <SearchByParlor />
  </div>
</header>
`;
